test/integration/daemonset/daemonset_test.go

```diff
@@ -447,6 +643,17 @@ func TestInsufficientCapacityNodeDaemonDoesNotLaunchPod(t *testing.T) {
 			t.Fatalf("Failed to create node: %v", err)
 		}
 
```

Comment:
Create another node here with enough resource to schedule, and the scheduler should schedule a pod on it.

Reply:
done.